api/src/pcapi/alembic/versions/20230202T135727_38f003447a1f_fixpricecategoryindex.py

```diff
@@ -0,0 +1,30 @@
+"""fixPriceCategoryIndex
```

Comment:
(micro-détail) 
```suggestion
"""Add ix_stock_priceCategoryId index
```

Ce sera plus clair dans l'historique Alembic.